test/00/simpleIntegrationTest.sh

```diff
@@ -30,59 +30,78 @@ pass()
 
 trap "fail" 1 2 3 15
 
-cat >input.tcl <<EOF
-source $here/test/assert.tcl
-
-# arrange for the following code to executed after Minsky has started.
-proc afterMinskyStarted {} {
-    # remove Destroy binding as it interferes with assert
-    bind . <Destroy> {}
-
-    minsky.load $here/examples/exponentialGrowth.mky
-    
-    getValue :y
-    assert {[value.value]==1}
-
-    set item "minsky.canvas.item"
-    findObject VarConstant
-    set a [\$item.value]
-    assert {[minsky.canvas.item.value]!=0}
-
-    # setting this flag prevents an extra reset occurring - used to
-    # distinguish between static plot updates and simulation
-    running 1 
-    minsky.reset
-    step
-    step  
-    getValue :y
-    assert {[value.value]>0}
-    if {\$a<0} {
-      assert {[value.value]>0 && [value.value]<1}
-    } else {
-      assert {[value.value]>0 && [value.value]>1}
-    }
-
-    set v [value.value]
-
-    findObject IntOp
-    \$item.toggleCoupled
-
-    minsky.reset
-    step
-    step
-    getValue :y
-    if {\$a<0} {
-      assert {[value.value]>0 && [value.value]<1}
-    } else {
-      assert {[value.value]>0 && [value.value]>1}
-    }
-    assert "\[value.value\]==\$v"
-
-    tcl_exit
-}
-
+cat >input.py <<EOF
+import sys
+sys.path.insert(0, '$here')
+from pyminsky import minsky, findObject
+
+# Step 1: Function to access value by ID
+def getValue(valueId):
+    try:
+        # Access valueId directly via minsky's interface
+        value = minsky.variableValues[valueId]
+        print(f"Found variable {valueId} with value: {value.value()}")
+        return value
+    except KeyError:
+        raise ValueError(f"Variable '{valueId}' not found in variableValues.")
+    except Exception as e:
+        raise ValueError(f"Error accessing variable '{valueId}': {e}")
+
+# Load the model
+minsky.load('$here/examples/exponentialGrowth.mky')
+
+# Step 2: Verify initial value of ':y'
+y_value = getValue(":y")
+assert y_value.value() == 1, f"Expected initial value of :y to be 1, but got {y_value.value()}"
+
+# Step 3: Find and check VarConstant
+findObject("VarConstant")
+var_constant_value = minsky.canvas.item.value()
+assert var_constant_value != 0, "VarConstant item value is unexpectedly 0."
+print(f"VarConstant value: {var_constant_value}")
+
+# Step 4: Advance simulation and verify updated ':y'
+def step():
+    """Advance the simulation by one step."""
+    minsky.step()
+
+# Run two simulation steps
+minsky.running = True
+minsky.reset()
+step()
+step()
+
+y_value_after_step = getValue(":y")
+assert y_value_after_step.value() > 0, "Value of :y after steps is not greater than 0."
```

Comment:
Original code was nuts. The assertion here is redundant with the assertions on lines 79 and 81.